alibi_detect/saving/tests/test_saving.py

```diff
@@ -808,6 +808,136 @@ def test_save_onlinefetdrift(data, tmp_path, seed):
     np.testing.assert_array_almost_equal(stats, stats_load, 4)
 
 
+@parametrize("detector", [MMDDriftOnline, LSDDDriftOnline])
+@parametrize_with_cases("data", cases=ContinuousData, prefix='data_')
+def test_save_multivariate_online_state(detector, data, backend, seed, tmp_path):
+    """
+    Test the saving (and loading) of multivariate online detectors' state when `save_state=True` is passed to
+    `save_detector`.
+    """
+    # Skip if backend not `tensorflow` or `pytorch`
+    if backend not in ('tensorflow', 'pytorch'):
+        pytest.skip("Detector doesn't have this backend")
```

Comment:
Is this due to some `keops` behaviour? Basically asking why skip here.

Reply:
The `test_saving` file cycles through all possible backends:

```python
backends = ['tensorflow', 'pytorch', 'sklearn']
if has_keops:  # pykeops only installed in Linux CI
    backends.append('keops')
backend = param_fixture("backend", backends)
```

We have to skip tests if the associated detector doesn't have that backend. In this case, online detectors do not have a `keops` backend.